app/controllers/tag_controller.rb

```diff
@@ -514,10 +514,12 @@ def stats
   end
 
   def comments
+    fetch_counts
```

Comment:
@billymoroney1 This method only returns the size of a `note`, `question`, `wiki` or `map`. Since we need the size of all of them and not just one, calling the method in `Tag#commets` is redundant.

Reply:
Also, when we click on the comments tab, the description, and name of the tag are not displayed. This is because we are not fetching the info about the tag itself in this method. This is how the tag info is gotten in `Tag#show`:

### Controller:

https://github.com/publiclab/plots2/blob/9198f100b36f4d8bcbedf20b79112868e8ce1278/app/controllers/tag_controller.rb#L78-L85

https://github.com/publiclab/plots2/blob/9198f100b36f4d8bcbedf20b79112868e8ce1278/app/controllers/tag_controller.rb#L151

### View: 

https://github.com/publiclab/plots2/blob/9198f100b36f4d8bcbedf20b79112868e8ce1278/app/views/tag/show.html.erb#L93

https://github.com/publiclab/plots2/blob/9198f100b36f4d8bcbedf20b79112868e8ce1278/app/views/tag/show.html.erb#L128-L130

Reply:
We could create a private method that fetches all of that data so we don't have to copy most of the code in `Tag#show` to `Tag#comments`